mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/openjpa/AbstractJPAMailboxMessage.java

```diff
@@ -505,7 +505,10 @@ public String toString() {
     @Override
     public List<MessageAttachment> getAttachments() {
         try {
-            return new MessageParser().retrieveAttachments(getFullContent());
+            return new MessageParser().retrieveAttachments(getFullContent())
```

Comment:
Do we really need the full content to get the Metadata ?

Reply:
Now, yes. I am not willing to add "a fetch type" for making this changeset even larger.